src/PAModelTests/ReadTransformTests.cs

```diff
@@ -11,18 +11,21 @@ namespace PAModelTests
 {
     [TestClass]
     public class ReadTransformTests
-    {   
+    {  
         [DataTestMethod]
-        [DataRow("GalleryTemplateNullChildren.msapp")]
-        public void AfterReadNullChildren(string filename)
+        [DataRow("GalleryTemplateNullChildren.msapp", false, false)]
+        [DataRow("TestStepWithInvalidScreen.msapp", false, true)]
```

Comment:
Is this a test for the RuleScreenIdWithoutScreen warning\error?

Reply:
Yes, a test .msapp that reproduced this error and is fixed now.